test/docker/fedoradev/install-packages.sh

```diff
@@ -16,10 +16,12 @@ dnf -y install /usr/bin/xargs
 while read -r file; do
     case $file in
         /*) printf "%s\n" "$file" ;;
-        *) printf "%s\n" {/usr,}/{,s}bin/"$file" ;;
+        *) printf "%s\n" {/usr,}/bin/"$file" ;;
     esac
-done |
-    xargs dnf --skip-broken -y install
+done | grep -v vpnc |
```

Comment:
Do we want to skip _everything_  matching the string vpnc, or just some specific package? Maybe there's room for making the grep more specific here?